ext/gd/libgd/gd.c

```diff
@@ -382,51 +382,6 @@ static float HWB_Diff (int r1, int g1, int b1, int r2, int g2, int b2)
 }
 
 
-#if 0
```

Comment:
Yes, please do not remove this code.
On May 30, 2015 3:46 PM, "Anatol Belski" <notifications@github.com> wrote:

> In ext/gd/libgd/gd.c
> <https://github.com/php/php-src/pull/1297#discussion_r31379054>:
>
> > @@ -382,51 +382,6 @@ static float HWB_Diff (int r1, int g1, int b1, int r2, int g2, int b2)
> >  }
> >
> >
> > -#if 0
>
> libgd is running it's own merging process, better ask @pierrejoye
> <https://github.com/pierrejoye> and @remicollet
> <https://github.com/remicollet>
>
> Thanks.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/php/php-src/pull/1297/files#r31379054>.
>